### PR TITLE
Feature/main page

### PR DESCRIPTION
**Pull request by kvnloughead**
Hopefully I am doing this right.   I have most of the main-page complete:  the scrollable menu bar, the grid of product-cards and (most of) the product-cards themselves.   Remaining to do:  create the special Sale card.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,5 +12,135 @@
     <li><img src="images/menu.svg" alt="menu icon"></li>
     <li><img src="images/search.svg" alt="search icon"></li>
   </ul>
+  <header class="header">
+    <ul class="header__category-list">
+      <li><a class="header__category header__category_active" href="">All</a></li>
```
Comment (momofcats) [inline index.html:17]: I think you need to write href="#" for all links

File: index.html [hunk 2]
```diff
@@ -12,5 +12,135 @@
...
+      <li><a class="header__category" href="">Fish</a></li>
+    </ul>
+  </header>
+  <main class="columns">
+    
+    <div class="column">
+      <a href="#" class="card">
+        <figure class="card__figure">
+          <figcaption class="card__category">Fruits</figcaption>
+          <img class="card__image" src="images/raspberries.png" alt="Some raspberries.">
```
Comment (momofcats) [inline index.html:34]: I would name this element card__caption

File: index.html [hunk 3]
```diff
@@ -12,5 +12,135 @@
...
+          <div class="card__ellipse"></div>
```
Comment (momofcats) [inline index.html:35]: I think, this div should be outside of the figure tag.

File: index.html [hunk 4]
```diff
@@ -12,5 +12,135 @@
...
+      <li><a class="header__category" href="">Bakery</a></li>
+      <li><a class="header__category" href="">Sandwiches</a></li>
+      <li><a class="header__category" href="">Dairy</a></li>
+      <li><a class="header__category" href="">Meat</a></li>
+      <li><a class="header__category" href="">Fish</a></li>
+    </ul>
+  </header>
+  <main class="columns">
+    
+    <div class="column">
```
Comment (momofcats) [inline index.html:30]: It seems like this is a good place for <ul>. Each card is a <li> item. But maybe I'm overthinking it.

File: blocks/header.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
+.header {
+  display: flex;
```
Comment (momofcats) [inline blocks/header.css:2]: Display: flex and flex-direction: column here is unnecessary.

File: images/+.svg [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (momofcats) [inline images/+.svg:1]: I don't understand whats going on here

File: index.html (hunk 2 above)
Comment (kvnloughead) [reply index.html:34]: Good idea.

File: index.html (hunk 3 above)
Comment (kvnloughead) [reply index.html:35]: Fixed.

File: index.html (hunk 4 above)
Comment (kvnloughead) [reply index.html:30]: Fixed

File: blocks/header.css (hunk 5 above)
Comment (kvnloughead) [reply blocks/header.css]: Ah, that was from when I was including to topmost icons in the header.   This actually makes me wonder though:  should we nest the navbar that you did along with the horizontally scrolling menu both inside the header?

File: images/+.svg (hunk 6 above)
Comment (kvnloughead) [reply images/+.svg:1]: This is the file for the little plus icon in the cards.

**Comment by kvnloughead** [comment]
I fixed the issues you brought up locally.   Closing this pull request.

**Pull request closed without merging**